htmlcss/index.html

```diff
@@ -147,6 +172,7 @@ <h1 class="page-header">
       <h2>
         Lesson 4: Intro to CSS
       </h2>
+      <button><a href="index_with_styling.html">Click here to see what CSS can do to this page!</a></button>
```

Comment:
here is some minor feedback

- can this be a straight up link instead of link + button?
- can you add more instructions about what students should look at or do after they click the link?